### ISSUE TITLE
no-find-dom-node: Does not detect named imported function

### ISSUE THREAD

**Issue by danez** [yannickcr/eslint-plugin-react#785]
The rule no-find-dom-node does not warn about this:

``` js
import { findDOMNode } from 'react-dom';

findDOMNode();
```

**Comment by Hypnosphi** [comment]
This makes the rule effectively useless =(

**Comment by Hypnosphi** [comment]
In fact, it only works if you call some method, like `findDOMNode(this).scrollIntoView` but doesn't otherwise

**Comment by daffy6537** [comment]
Hi，there is a question, I want create a Animation component container, need to  I want to use the findDOMNode Api to continuely modify the container's height style, But the plugin chaimed me that Line 37:  Do not use findDOMNode  react/no-find-dom-node, how can I deal with it?

**Comment by Hypnosphi** [comment]
Two options I see:

1. Try using refs instead
2. `// eslint-ignore-line react/no-find-dom-node`
